lib/ansiblelint/utils.py

```diff
@@ -241,7 +241,19 @@ def _taskshandlers_children(basedir, k, v, parent_type):
         if 'include' in th:
             append_children(th['include'], basedir, k, parent_type, results)
         elif 'include_tasks' in th:
-            append_children(th['include_tasks'], basedir, k, parent_type, results)
+            append_children(
+                th['include_tasks'].get(
+                    'file',
+                    th['include_tasks']
+                ),
+                basedir,
+                k,
+                parent_type,
+                results
+            )
+                append_children(th['include_tasks']['file'], basedir, k, parent_type, results)
+            else:
+                append_children(th['include_tasks'], basedir, k, parent_type, results)
```

Comment:
There's something wrong with indentation in these lines.

Reply:
That suggested change was supposed to hit more lines than one but at the time GH didn't support multiline comments.